component/remote/abs.go

```diff
@@ -50,13 +50,13 @@ func (a *AbsApolloConfig) SyncWithNamespace(namespace string, appConfigFunc func
 	apolloConfig, err := http.RequestRecovery(appConfig, c, &callback)
 	if err != nil {
 		log.Errorf("request %s fail, error:%v", urlSuffix, err)
-		return nil
+		return nil, err
 	}
 
 	if apolloConfig == nil {
 		log.Debug("apolloConfig is nil")
-		return nil
+		return nil, err
```

Comment:
The err is always nil here.
```suggestion
		return nil, nil
```